source/administration/config-settings.rst

```diff
@@ -2149,9 +2149,9 @@ Includes three configuration keys: `ImageProxyType`, `ImageProxyURL` and `ImageP
 
 .. _documentation: https://docs.mattermost.com/administration/image-proxy
```

Comment:
@amyblais Is this formatted correctly?

Reply:
I don't know how to format differently.

Reply:
@amyblais I'm not sure either, maybe Eric does?

Reply:
@esethna Would you know how to format the link properly?